lib/rules/no-unexpected-multiline.js

```diff
@@ -24,11 +24,13 @@ module.exports = function(context) {
      */
     function checkForBreakAfter(node, msg) {
         var paren = context.getTokenAfter(node);
+        var beforeParen = node;
         while (paren.value === ")") {
+            beforeParen = paren;
             paren = context.getTokenAfter(paren);
```

Comment:
I think this variable being called `paren` is confusing, if I'm reading it right it ends up being the next node *after* the paren?